pretext/Lists/ListMethods.ptx

```diff
@@ -1,49 +1,16 @@
 <?xml version="1.0"?>
 <section xml:id="lists_list-methods">
   <title>List Methods</title>
-  <p>The dot operator can also be used to access built-in methods of list objects.
-            <c>append</c> is a list method which adds the argument passed to it to the end of
-            the list. Continuing with this example, we show several other list methods.  Many of them are
-            easy to understand.</p>
-  <program xml:id="ch4_sec10_p1" interactive="activecode" language="python">
-    <input>
-mylist = []
-mylist.append(5)
-mylist.append(27)
-mylist.append(3)
-mylist.append(12)
-print(mylist)
 
-mylist.insert(1, 12)        # Insert 12 at pos 1, shift other items up
-print(mylist)
-print(mylist.count(12))     # How many times is 12 in mylist?
-
-print(mylist.index(3))      # Find index of first 3 in mylist
-print(mylist.count(5))
-
-mylist.reverse()
-print(mylist)
-
-mylist.sort()
-print(mylist)
+<p> The dot operator can also be used to access built-in methods of list objects.
+    As you remember, unlike strings, lists are mutable. As a consequence, list methods can have different behaviors than string methods. Let's look at these.</p>
+<ul>
+<li> <term>Return Type Methods:</term> These methods work the same way as string methods: they give us a value but keep the original list the same.</li>
+<li> <term>Mutating Methods:</term> These methods modify the list but do not return anything Because of this, we shouldn't use these methods on the right hand side of assignment.</li>
```

Comment:
I would take this a step further, and just say that we shouldn't assign anything to these at all

Reply:
done